groundzero/rebuild_from_ground_zero.sh

```diff
@@ -1,32 +1,23 @@
 cd /app/groundzero/
-psql "sslmode=verify-ca sslrootcert=/root/.postgresql-rw/root.crt sslcert=/root/.postgresql-rw/postgresql.crt sslkey=/root/.postgresql-rw/postgresql.key hostaddr=$DB_HOST_RW user=rmuser password=password dbname=$DB_NAME_RW" -f destroy_schemas.sql
+
+PSQL_CONNECT_WRITE_MODE="sslmode=verify-ca sslrootcert=/root/.postgresql-rw/root.crt sslcert=/root/.postgresql-rw/postgresql.crt sslkey=/root/.postgresql-rw/postgresql.key hostaddr=$DB_HOST_RW user=rmuser password=password dbname=$DB_NAME_RW"
+psql "$PSQL_CONNECT_WRITE_MODE" -f destroy_schemas.sql
 
 rm actionv2.sql
 rm casev2.sql
 rm uacqid.sql
+rm ddl_version.sql
 
-curl https://raw.githubusercontent.com/ONSdigital/census-rm-ddl/master/groundzero_ddl/actionv2.sql -o actionv2.sql
-curl https://raw.githubusercontent.com/ONSdigital/census-rm-ddl/master/groundzero_ddl/casev2.sql -o casev2.sql
-curl https://raw.githubusercontent.com/ONSdigital/census-rm-ddl/master/groundzero_ddl/uac_qid.sql -o uacqid.sql
-
-echo "begin transaction;" > header_footer_temp.txt
-cat actionv2.sql >> header_footer_temp.txt
-echo "commit transaction;" >> header_footer_temp.txt
-rm actionv2.sql
-mv header_footer_temp.txt actionv2.sql
-
-echo "begin transaction;" > header_footer_temp.txt
-cat casev2.sql >> header_footer_temp.txt
-echo "commit transaction;" >> header_footer_temp.txt
-rm casev2.sql
-mv header_footer_temp.txt casev2.sql
+for SCRIPT_NAME in actionv2.sql casev2.sql uacqid.sql ddl_version.sql
+do
+  rm $SCRIPT_NAME
```

Comment:
Probably don't need to remove this first as it's removed earlier on?

Reply:
@liamtoozer I actually meant to delete those first `rm`s so the script names are only has to have the script names in one place, I'll fix that